src/ndarray/ndarray.cc

```diff
@@ -39,11 +39,13 @@
 #include "../operator/tensor/matrix_op-inl.h"
 #include "../operator/tensor/init_op.h"
 #include "../operator/nn/mkldnn/mkldnn_base-inl.h"
+#include "../engine/engine_impl.h"
```

Comment:
is this still needed?

Reply:
nope, seems like leftover from merge, removed now.

Reply:
I think I missed a rebase change, sorry, thanks for catching it!